provision-vm-service/list-providers.asl

```diff
@@ -12,8 +12,9 @@
       },
       "Parameters": {
         "API_URL.$": "$$.Execution._manageiq_api_url",
-        "PROVIDER_TYPE.$": "ManageIQ::Providers::Vmware::InfraManager",
-        "VERIFY_SSL.$": "$.dialog.dialog_verify_ssl"
+        "VERIFY_SSL.$": "$.dialog.dialog_verify_ssl",
+
+        "PROVIDER_TYPE": "ManageIQ::Providers::Vmware::InfraManager"
```

Comment:
This type of change is just to visually split those parameters that are for API login vs those needed for the "work"

Reply:
Also this specific line was interesting.  Previously we had `"PROVIDER_TYPE.$"`, but the ASL parser in VSCode didn't like that because technically, this value had no interpolation nor intrinsic function on the right hand side.  So, I removed the `.$`.  This may be something we also want to enforce in floe, or perhaps we want to remove this error in the ASL VSCode extension, instead maybe showing a warning.

Reply:
:+1: agreed if this is going to be hard coded then it shouldn't end in `.$`
Maybe we allow this to be dynamic in the future but clone-template&friends are vmware specific so makes sense to keep it just vmware right now